### ISSUE TITLE
MP immune system process

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#14262]
Any reason why this is not under "system process"?

**Comment by ValWood** [comment]
Hmm, I see
Comments
Note that this term is a direct child of 'biological_process ; GO:0008150' because some immune system processes are types of cellular process (GO:0009987), whereas others are types of multicellular organism process (GO:0032501). 

but isn't that true of any system process?

**Comment by pgaudet** [comment]
That seems odd to me. 
cellular process (GO:0009987) and multicellular organism process (GO:0032501) are not mutually exclusive; ie all (or at least most) processes that occur in multicellular organisms do occur at the cellular level. I would say that the immune cells, even when they float around in the blood (or whereever they may be), are part of a *system* that acts in a coordinated manner. 

On the other hand some types of 'multicellular organism process ' and 'unicellular organism' processes may be. Isn't it? 

If you remove the 'regulation of immune system process' (which viruses/bacteria can do), and remove eukaryotes, there are only few EXP annotations left: 
UniProtKB:P75960	protein-lysine deacetylase/desuccinylase	defense response to virus	GO:0051607	EcoCyc	NCBITaxon:83333	IMP	PMID:25516616	20161221
UniProtKB:Q8GED7	Cyclo(L-leucyl-L-phenylalanyl) synthase	antibacterial peptide biosynthetic process	GO:0002780	UniProt	NCBITaxon:1971	IDA	PMID:19430487	20130617
UniProtKB:B8X8Z0	Endoribonuclease ToxN	defense response to virus	GO:0051607	UniProt	NCBITaxon:29471	IDA	PMID:19124776	20150227
UniProtKB:Q2YLF3	4-hydroxyproline epimerase	macrophage activation	GO:0042116	CACAO	NCBITaxon:359391	IDA	PMID:23720774	20141123
UniProtKB:A4VWH9	Enolase	antibacterial humoral response	GO:0019731	CAFA	NCBITaxon:391295	IDA	PMID:19848587	20160712
UniProtKB:Q46901	Cascade subunit A	defense response to virus	GO:0051607	EcoCyc	NCBITaxon:83333	IMP	PMID:18703739	20110725
UniProtKB:Q46899	Cascade subunit C	defense response to virus	GO:0051607	EcoCyc	NCBITaxon:83333	IMP	PMID:18703739	20110725
UniProtKB:Q46898	Cascade subunit D	defense response to virus	GO:0051607	EcoCyc	NCBITaxon:83333	IDA	PMID:18703739	20110725
UniProtKB:Q46897	pre-CRISPR RNA endonuclease	defense response to virus	GO:0051607	EcoCyc	NCBITaxon:83333	IMP	PMID:18703739	20110725
UniProtKB:P76632	Cascade subunit B	defense response to virus	GO:0051607	EcoCyc	NCBITaxon:83333	IMP	PMID:18703739	20110725
UniProtKB:P52126	CP4-57 prophage; putative helicase YfjK	defense response to virus	GO:0051607	EcoCyc	NCBITaxon:83333	IMP	PMID:25224971	20141009
UniProtKB:P38036	Cas3 nuclease/helicase	defense response to virus	GO:0051607	EcoCyc	NCBITaxon:83333	IMP	PMID:21699496	20110706
UniProtKB:P38036	Cas3 nuclease/helicase	defense response to virus	GO:0051607	EcoCyc	NCBITaxon:83333	IDA	PMID:18703739	20080910
UniProtKB:P52127	CP4-57 prophage; anti-bacteriophage protein	defense response to virus	GO:0051607	EcoCyc	NCBITaxon:83333	IMP	PMID:25224971	20141009

The terms there should not be under immune response. 

Also if you remove Metazoa and look at what's annotated in lower eukaryotes: 
- Dicty has a few IBA/IEA (probably by mistake)
- There are a number of reactome annotations to bacteria and fungal proteins, I dont think these would be missed (right @deustp01 ?) 

One question I have is for @tberardini : a few plant proteins are annotated to 'systemic acquired resistance'; is this a multicellular organism process ? (it sounds like a dumb question; if the plant is multicellular then it should be a multicellular organism process.... ) 

Thanks, Pascale

**Comment by tberardini** [comment]
@pgaudet : yes, 'systemic acquired resistance' is a multicellular organism process.  Classic paper:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2634024/

**Comment by cmungall** [comment]
The challenge may be with single-celled organisms rather than cellular 
processes per se. In CARO, anatomical system is multicellular, and GO 
follows suit:

id: GO:0003008
name: system process
namespace: biological_process
def: "A multicellular organismal process carried out by any of the 
organs or tissues in an organ system. An organ system is a regularly 
interacting or interdependent group of organs or tissues that work 
together to carry out a biological objective." [GOC:mtg_cardio]
synonym: "organ system process" EXACT []
is_a: GO:0032501 ! multicellular organismal process

So this means that CRISPR in bacteria can't be a system process, by this 
definition

At the very least, it could be said that GO's nomenclature is confusing. 
This class should probably be renamed something like 'anatomical system 
process' or even the ugly 'gross anatomical system process'.  But it's 
really not clear how useful the class is. Or how useful it is having a 
class for immune processes that are restricted to those that occur in 
multi-cellular systems.

oh, and it looks like CRISPR-cas isn't even under immune system process:

  GO:0008150 ! biological_process
   is_a GO:0050896 ! response to stimulus
    is_a GO:0006950 ! response to stress
     is_a GO:0006952 ! defense response
      is_a GO:0099046 ! clearance of foreign intracellular nucleic acids
       is_a GO:0099048 ! CRISPR-cas system ***

The text definition uses the genus 'adaptive immune response':

id: GO:0099048
name: CRISPR-cas system
namespace: biological_process
def: "An **adaptive immune response** of bacteria that serves to clear 
host cells of foreign DNA and RNA. It has three distinct stage: 
acquisition of foreign DNA by integration into CRISPR loci in the host 
chromosome, CRISPR RNA (crRNA) biogenesis, and target interference. 
CISPR stands for Clustered Regularly Interspaced Short Palindromic 
Repeat, which describes the nature of the loci." [PMID:23495939]
is_a: GO:0099046 ! clearance of foreign intracellular nucleic acids

Which is terminologically inconsistent with this GO class that bakes in 
assumptions about the immune mechanism:

id: GO:0002250
name: adaptive immune response
namespace: biological_process
def: "An immune response mediated by cells expressing specific receptors 
for antigen produced through a somatic diversification process, and 
allowing for an enhanced secondary response to subsequent exposures to 
the same antigen (immunological memory)." [GO_REF:0000022, GOC:add, 
GOC:mtg_15nov05, ISBN:0781735149]
comment: This term was added by GO_REF:0000022.
synonym: "acquired immune response" EXACT [ISBN:068340007X]
synonym: "immune memory response" EXACT [GOC:add]
xref: Wikipedia:Adaptive_immune_system
is_a: GO:0006955 ! immune response

TL;DR - there is a lot of implicit taxonomic assumptions built in here 
that will take a while to unpack. We need to axiomatize this as much as 
possible to detect these issues and automate the classification.

**Comment by ukemi** [comment]
Seems to me that the comment here explains why this is ok. In some cases, immune system processes are cellular processes and in some cases they are multicellular processes. Therefore even though immune system process has that name, it does not fit the definition of system process.

**Comment by ValWood** [comment]
OK close?

**Comment by pgaudet** [comment]
From PAINT's perspective, it is always useful to distinguish the multicellular organism processes vs the single organism processes. 

how about creating a term 'immune response' to cover adaptive immune response, that occurs in all organisms, and move 'immune system process' under 'system process'? That seems more correct at least evolutionarily. 

@thomaspd do you have an opinion on this one ? 

Thanks, Pascale

**Comment by ValWood** [comment]
hi, @pgaudet  is this might be relevant to current work. Added multi-species label

**Comment by pgaudet** [comment]
@addiehl thoughts?

**Comment by pgaudet** [comment]
There is a comment that explains why immune system process is not under "system process": 
"Note that this term is a direct child of 'biological_process ; GO:0008150' because some immune system processes are types of cellular process (GO:0009987), whereas others are types of multicellular organism process (GO:0032501)." 

I am closing this - @ValWood reopen if you have a suggestion as to how to fix this.
